autoload/ale/linter.vim

```diff
@@ -174,26 +174,25 @@ function! ale#linter#Define(filetype, linter) abort
     call add(s:linters[a:filetype], l:new_linter)
 endfunction
 
-function! ale#linter#GetAll(filetype) abort
-    if a:filetype ==# ''
-        " Empty filetype? Nothing to be done about that.
-        return []
-    endif
+function! ale#linter#GetAll(filetypes) abort
+    let l:combined_linters = []
 
-    if has_key(s:linters, a:filetype)
-        " We already loaded the linter files for this filetype, so stop here.
-        return s:linters[a:filetype]
-    endif
+    for l:filetype in a:filetypes
+        " Haven't we loaded the linter files for this filetype yet?
+        if !has_key(s:linters, l:filetype)
+            " So load it
+            execute 'silent! runtime! ale_linters/' . l:filetype . '/*.vim'
 
-    " Load all linters for a given filetype.
-    execute 'silent! runtime! ale_linters/' . a:filetype . '/*.vim'
+            " Still don't have the linter files? There must be occured an error
+            if !has_key(s:linters, l:filetype)
+                return []
+            endif
```

Comment:
Done. Now I'm wondering if there's a way to retry to load a linter that failed to load at first but give up after too many failures...